system/modules/isotope/library/Isotope/Model/Payment/Payone.php

```diff
@@ -168,4 +164,22 @@ function(&$option) {
 
         return $objTemplate->parse();
     }
+
+    /**
+     * Escape the parameter value so it's correctly handled by Payone
+     *
+     * @param string $value
+     *
+     * @return string
+     */
+    protected function escapeParameterValue($value)
+    {
+        $value = strip_insert_tags(strip_tags($value));
+        $value = \StringUtil::restoreBasicEntities($value);
+        $value = html_entity_decode($value);
+        $value = str_replace("\xC2\xA0", ' ', $value); // strip &nbsp;
+        $value = str_replace('"', '\'', $value); // replace double quotes with single quotes
```

Comment:
Ah, that's what I use `StringUtil::specialchars` usually. One or the other would be ok, but why not use `\"` then?